### PR TITLE
fix typo in arrow-parens.md

### PR DESCRIPTION
**Pull request by wwwillchen**
**What is the purpose of this pull request? (put an "X" next to item)**

[x ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

**What changes did you make? (Give an overview)**

Fixed a typo

### CODE REVIEW

**Comment by gyandeeps** [comment]
LGTM :+1: 
Thanks for contributing.

File: docs/rules/arrow-parens.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ String options are:
 
 Object properties for variants of the `"as-needed"` option:
 
-* `"requireForBlockBody": true` modifies the as-needed rule in order to require parens if the function body is in an intructions block (surrounded by braces).
+* `"requireForBlockBody": true` modifies the as-needed rule in order to require parens if the function body is in an instructions block (surrounded by braces).
```
Comment (platinumazure) [inline docs/rules/arrow-parens.md]: I'm a bit confused on what this is actually trying to say, to be honest. Is this saying if the arrow body is a block with statements in it? If so, I think I would prefer "statement block" or "block with statements"...

Comment (platinumazure) [inline docs/rules/arrow-parens.md]: Lol, guess I was too slow. Thanks @wwwillchen for contributing!

Comment (gyandeeps) [inline docs/rules/arrow-parens.md]: lol yeah... he was trying to fix the spelling. :smile:

**Pull request merged**